src/main/java/rearth/oritech/init/Config.java

```diff
@@ -60,6 +60,12 @@ public class Config {
     public int endPortalRequiredSpeed = 10000;
     public int netherPortalRequiredSpeed = 5000;
     public int blackHoleRequiredSpeed = 15000;
+
+    @SectionHeader("blackHole")
+    public boolean enableBlackHoleBlock = true;
```

Comment:
I don't think we actually need this part. The item is not sustainable in survival (e.g. on servers), and in creative I believe the players should just be able to do what they want. The item exists just for players to be able to build it in creative.